assets/wizards/advertising/views/providers/index.js

```diff
@@ -57,6 +94,28 @@ const Providers = ( { services, toggleService } ) => {
 					},
 				} }
 			/>
+			{ isOnboarding && (
+				<Modal
+					title={ __( 'Google Ad Manager Setup', 'newspack-ads' ) }
+					onRequestClose={ () => setIsOnboarding( false ) }
+				>
+					<GAMOnboarding
+						onUpdate={ data => setNetworkCode( data.networkCode ) }
+						onSuccess={ () => {
+							fetchAdvertisingData();
+							setIsOnboarding( false );
+						} }
+					/>
+					<Card buttonsCard noBorder className="justify-end">
+						<Button isSecondary disabled={ inFlight } onClick={ () => setIsOnboarding( false ) }>
+							{ __( 'Cancel', 'newspack' ) }
+						</Button>
+						<Button isPrimary disabled={ inFlight } onClick={ () => updateGAMNetworkCode() }>
```

Comment:
This button should be disabled until the network code and credentials are supplied.

Reply:
4850c87fc7f811fb799c87b6b22f5fa60f22aeeb disables the button if the network code is not filled, but the service account credential still resolves the modal because it provides a network code.